RecoLocalTracker/SiStripClusterizer/plugins/SiStripClusters2ApproxClustersv1.cc

```diff
@@ -0,0 +1,51 @@
+#include "RecoLocalTracker/SiStripClusterizer/interface/SiStripClusters2ApproxClustersv1.h"
+#include <iostream>
+
+
+SiStripClusters2ApproxClustersv1::SiStripClusters2ApproxClustersv1(const edm::ParameterSet& conf){
+   inputClusters = conf.getParameter< edm::InputTag >("inputClusters");
+   clusterToken = consumes< edmNew::DetSetVector< SiStripCluster > >(inputClusters);
+
+   produces< edmNew::DetSetVector< SiStripApproximateClusterv1 > >(); 
+
+}
+
+void SiStripClusters2ApproxClustersv1::produce(edm::Event& e, edm::EventSetup const&){
+  std::unique_ptr<edmNew::DetSetVector< SiStripApproximateClusterv1 > > result(new edmNew::DetSetVector< SiStripApproximateClusterv1 > );
+
+  edm::Handle<edmNew::DetSetVector< SiStripCluster >> clusterCollection;
+  e.getByToken(clusterToken, clusterCollection);
```

Comment:
```suggestion
  edm::Handle<edmNew::DetSetVector< SiStripCluster >> clusterCollection = e.getHandle(clusterToken_)
```
as suggested at https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideEDMGetDataFromEvent#getHandle

or IIUC even better:

```C++
const auto& clusters = e.get(clustersToken_);
```